async.md

```diff
@@ -218,3 +220,100 @@ Example
 			resp.error("caught: "+reason.message);
 		})
 ~~~
+
+# Messaging
+
+This class allows for the execution of async ClearBlade messaging functions provide an MQTT client that will work asynchronously. You can connect to the internal ClearBlade broker or an external broker. 
+
+## MQTT.Message(payload)
+
+The message represents an MQTT message.
+
+* @param {string} payload
+* @constructor
+* @property {string} payload
+* @property {number} qos
+* @property {boolean} retain
+* @property {boolean} duplicate
+
+Example
+
+~~~javascript
+	var msg = new MQTT.Message()
```

Comment:
Need a payload here. should be something like `var msg = new MQTT.Message("my message")`